server/.env.example

```diff
@@ -1 +1,6 @@
 PORT=80
+PGHOST=localhost
```

Comment:
Add the PG env elements (lines 2 to 6) inside the code block of step 2 under Setup in the README

Reply:
I went ahead and refactored the `.env` files to a better pattern. This way unless you are building to deploy to production you dont need to touch any of the `.env` files and we can customize them for tests, development, production, etc